src/core/diags/README.md

```diff
@@ -76,6 +76,7 @@ status 0x00
 ### diag send \<packets\> \<length\>
 
 Transmit a fixed number of packets with fixed length.
+Length parameter has to be in range [3;127].
```

Comment:
Comma is used instead of semicolon for expressing ranges in this file.

Reply:
Fixed.